docs/conf.py

```diff
@@ -25,11 +25,8 @@
 # -- General configuration ------------------------------------------------
 import datetime
 import os
-
-# Get configuration information from setup.cfg
 from configparser import ConfigParser
 
-# Sphinx gallery
```

Comment:
How come this comment got cut?

Reply:
I found a comment "Sphinx gallery" before an import "sphinx_gallery" pretty useless...